src/redux/reducers/index.ts

```diff
@@ -2,11 +2,9 @@
 import { combineReducers } from "redux";
 
 import tweetsSlice from "./tweets";
-import { userSlice } from "./user";
+import userSlice from "./user";
 
 export const rootReducer = combineReducers({
-  tweets: tweetsSlice.reducer,
-  user: userSlice.reducer,
+  tweets: tweetsSlice,
+  user: userSlice,
```

Comment:
```suggestion
  tweets: tweetsReducer,
  user: userReducer,
```

at that place we have reducers not slice!